metricbeat/docs/autodiscover-hints.asciidoc

```diff
@@ -35,6 +35,16 @@ The time interval for metrics retrieval, ie: 10s
 
 Metrics retrieval timeout, default: 3s
 
+[float]
+===== `co.elastic.metrics/username`
+
+The username to use for authentication
+
+[float]
+===== `co.elastic.metrics/password`
+
+The password to use for authentication
```

Comment:
I think we should put a note here advising not to use plain text passwords here, but references to ENV vars in the Metricbeat container, or passwords stored in keystore. Long term I think we should implement a way to allow referencing passwords from k8s secrets